pkg/timeutils/json.go

```diff
@@ -24,3 +25,12 @@ func FastMarshalJSON(t time.Time) (string, error) {
 	}
 	return t.Format(JSONFormat), nil
 }
+
+func AcceptedFormats() []string {
+	AcceptedFormats := []string{
```

Comment:
I actually wonder whether this should be a package level variable, wdyt @LK4D4 @unclejack ? Would prevent allocating this for each call of `setTime`.